commands.sh

```diff
@@ -70,10 +71,13 @@ bash scripts/download.sh
 #   - input the overall prompt+new contents into an LLM (e.g., Claud Sonet 3.5+) API or Web console to get answer.
 #   - copy the answer (in RAW format!!!!!) and save to a file, named type.txt under the mag/ folder.
 
-# 2. run autog, and explanation of these arguments
+# 2. Option 1: run autog (manually call LLM), and explanation of these arguments
 #   mag             -> name of the --dataset argument
 #   /data/datasets  -> path of the folder that store data, .e.g, /data/datasets
 #   autog-s         -> the method to run the model
 #   type.txt        -> the name of file to save analysis results from LLM. Since we manually create this, this file is not used.
 #   venue           -> Name of the task to fit the solution
 python -m main.autog mag ./data/datasets/mag autog-s type.txt venue
+
+# 2. Option 2: run autog2 (automatically interact with LLM for single or multiple rounds)
```

Comment:
leave a comment to explain that to run this command, need to set up environment variables for AWS bedrock access.